gradboost_pv/models/training.py

```diff
@@ -100,13 +110,34 @@ def run_experiment(
     X_train, y_train = X.loc[X.index < "2021-01-01"], y.loc[y.index < "2021-01-01"]
     X_test, y_test = X.loc[X.index >= "2021-01-01"], y.loc[y.index >= "2021-01-01"]
 
+    # Xy = QuantileDMatrix(X_train, y_train)
```

Comment:
how come the comments?

Reply:
I was going off of an example that used that way of training the model. I'll be removing it since the way it is now works, and is less changes than before.